README.md

```diff
@@ -1,48 +1,68 @@
+### Development Version
 
-![hover screenshot](https://raw.githubusercontent.com/tomv564/LSP/master/docs/images/screenshot-hover.png)
+* Clone this repository into your Packages directory.
+* Open the command palette and run `Package Control: Satisfy Dependencies`.
 
-As-you-type diagnostics with support for code fixes (`F4` to select, `super+.` to trigger actions)
 
-![diagnostics screenshot](https://raw.githubusercontent.com/tomv564/LSP/master/docs/images/screenshot-diagnostics-action.png)
+## Getting started
 
-## Installing
+Copy the configuration for a <a href="#available_languages">language</a>. 
 
-Normal installation: Install the **LSP** package through Package Control in Sublime Text.
+Open `LSP.sublime-settings` and paste the configuration in the `clients` section.
 
-To run latest master:
-1. Clone this repository into your Packages directory
-2. Run `Package Control: Satisfy Dependencies` in Sublime
+Open a document supported by the language server. LSP should report the language server starting in the status bar. 
 
-## Configuration
+If the server doesn't start. Open the command palette and run `LSP: Enable Language Server` then select the server. If you still have any problems, see [troubleshooting](https://lsp.readthedocs.io/en/latest/#troubleshooting) problems or start a discussion in gitter [chat](https://gitter.im/SublimeLSP).
 
-1. Install a language server for a language of your choice
-2. Run `LSP: Enable Language Server` from the Command Palette to enable your server.
-  * A configuration can be added under `clients` in `Preferences: LSP Settings`
-3. Open a document supported by this language server.
-4. LSP should report the language server starting in the status bar.
+Documentation is available at [LSP.readthedocs.io](https://LSP.readthedocs.io).
 
-Documentation is available at [LSP.readthedocs.io](https://LSP.readthedocs.io) or [in the docs directory](https://github.com/tomv564/LSP/blob/master/docs/index.md)
 
-## Troubleshooting
+## Available Languages <a name="available_languages"></a>
+* [Bash](https://lsp.readthedocs.io/en/latest/#bash)
+* [C/C++](https://lsp.readthedocs.io/en/latest/#clangd)
+* [CSS/LESS/SASS (SCSS only)](https://lsp.readthedocs.io/en/latest/#css)
+* [Dart](https://lsp.readthedocs.io/en/latest/#dart)
+* [Flow (JavaScript)](https://lsp.readthedocs.io/en/latest/#flow)
+* [Go](https://lsp.readthedocs.io/en/latest/#go)
+* [HTML](https://lsp.readthedocs.io/en/latest/#html)
+* [Java (Eclipse)](https://lsp.readthedocs.io/en/latest/#java)
+* [Java (IntelliJ)](https://lsp.readthedocs.io/en/latest/#intellij)
+* [JavaScript/TypeScript](https://lsp.readthedocs.io/en/latest/#typescript)
+* [JSON](https://lsp.readthedocs.io/en/latest/#json)
+* [Julia](https://lsp.readthedocs.io/en/latest/#julia)
+* [Kotlin](https://lsp.readthedocs.io/en/latest/#kotlin)
+* [PHP (Felix Becker)](https://lsp.readthedocs.io/en/latest/#php)
+* [PHP (Intelephense)](https://lsp.readthedocs.io/en/latest/#intelephense)
```

Comment:
For all the languages that have more than one server, can just link to eg. "PHP" instead of listing the servers? Keeps the list shorter and prevents us from having to prefer one server over the other.

Reply:
If I understood correctly. The README.md file will have just one link to the language.
```jsonc
[Kotlin](https://lsp.readthedocs.io/en/latest/#kotlin)
[PHP](https://lsp.readthedocs.io/en/latest/#php) // <- 
...
```

Than in `index.md`. I could list all the implementations, and the user could select what he wants.
```
### Kotlin <a name="kotlin"></a>
...

### PHP <a name="php"></a>
Implementations:
* [PHP (Felix Becker)](https://lsp.readthedocs.io/en/latest/#felix_becker)
* [PHP (Intelephense)](https://lsp.readthedocs.io/en/latest/#intelephense)

#### PHP (Felix Becker)<a name="felix_becker"></a>
...


#### PHP (Intelephense)<a name="intelephense"></a>
...
```

Does it solve the 'prefer one server over another'?

Reply:
Yes, keep it simple on the readme, show the options in the docs!